### PR TITLE
update staging

### PR DESCRIPTION
**Pull request by Jonath-z**

### CODE REVIEW

File: src/components/InputBar.jsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        onChange={(e) => setCurrentTask(e.target.value)}
+        defaultValue={currentTask}
+        value={currentTask}
+        placeholder="New task"
+        className="w-full border-gray-300 shadow-xl shadow-orange-200 py-3 px-5 rounded-3xl outline-orange-300"
+      />
+      <button
+        onClick={(e) => {
+          e.preventDefault();
+          if (currentTask !== "")
```
Comment (musayann) [inline src/components/InputBar.jsx]: I would move this function outside the render to the upper part of the component to increase code readability

File: src/components/TodoCard.jsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        <input
+          type="checkbox"
+          checked={todo.done}
+          onChange={() => dispatch(achieveTodo(todo))}
+        />
+        <p
+          contentEditable={true}
+          suppressContentEditableWarning={true}
+          onBlur={(e) =>
+            dispatch(updateTask({ ...todo, task: e.target.textContent }))
```
Comment (musayann) [inline src/components/TodoCard.jsx]: I would avoid spreading this todo object, and just pass in. two parameters one could be the old todo, and the second would be the change object.

File: src/utils/storage.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
+function setItem(key, value) {
+  localStorage.setItem(key, JSON.stringify(value));
```
Comment (musayann) [inline src/utils/storage.js:2]: did you use this file because you were dealing with `localStorage` ?

Comment (Jonath-z) [reply src/utils/storage.js:2]: Yes that's the reason

**Pull request merged**